colpali_engine/utils/dataset_transformation.py

```diff
@@ -210,6 +210,21 @@ def __call__(self, *args, **kwargs):
         return dataset
 
 
+class TestSetFactoryBEIR:
+    def __init__(self, dataset_path):
+        self.dataset_path = dataset_path
+
+    def __call__(self, *args, **kwargs):
```

Comment:
2 questions here:
1. Is there a reason for keeping `*args, **kwargs`?
2. Can we expose `split: str = "test"` in the `__call__` args?

```suggestion
    def __call__(self, *args, **kwargs):
```